applications/trilinos_application/trilinos_space.h

```diff
@@ -492,7 +492,7 @@ class TrilinosSpace
 
     //***********************************************************************
 
-    static void GatherValues(const VectorType& x, const std::vector<std::size_t>& IndexArray, double* pValues)
+    static void GatherValues(const VectorType& x, const std::vector<int>& IndexArray, DataType* pValues)
```

Comment:
It was done because Epetra could not convert the type if the "int" was not supplied. In fact if this method is compiled is giving a compilation error with std::size_t

Reply:
How it was compiling until now? Nobody was using it?

Reply:
I also agree with the DataType